docs/src/prototypes/popups/GridImagePicker/GridImagePicker.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+
+import { Grid, Input, gridBehavior } from '@stardust-ui/react'
+import GridImagePickerItem, { GridPickerItemProps } from './GridImagePickerItem'
```

Comment:
I'm going to introduce a tslint rule for imports, but for now, let's just arrange them

```suggestion
import { Grid, Input, gridBehavior } from '@stardust-ui/react' // stardust is on top because it's namespaced package
import * as _ from 'lodash' // other goes by module name
import * as React from 'react'

import GridImagePickerItem, { GridPickerItemProps } from './GridImagePickerItem'
```

Reply:
is it some best practice of how to arrange imports?

Reply:
Yep, imports in SUIR are arranged in the same manner